src/movie-components/movie-card-component/movie-card-component.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+
+function MovieCard (props) {
+  return (
+    <div className='movie-card'>
+      <h1>{props.title}</h1>
+      <img src={props.poster_path}/>
+      <div>
+        <h2>Genre</h2>
+        <p>*Insert Genre Here*</p>
```

Comment:
Can start planning on how to incorporate the genre data once APIs are being used.